packages/cli/src/databases/entities/ExecutionEntity.ts

```diff
@@ -49,7 +50,7 @@ export class ExecutionEntity {
 	@Column({ type: datetimeColumnType, nullable: true })
 	stoppedAt: Date;
 
-	@Column(datetimeColumnType)
+	@DeleteDateColumn({ type: datetimeColumnType, nullable: true })
```

Comment:
are we really benefiting from this decorator?

Reply:
[`@DeleteDateColumn`](https://orkhan.gitbook.io/typeorm/docs/decorator-reference#deletedatecolumn) is what enables `softDelete` to work, which is used in many places:

<img width="398" alt="Capture 2023-09-18 at 14 48 19@2x" src="https://github.com/n8n-io/n8n/assets/44588767/93e445dc-9194-49ae-bb26-4a0a0486f559">